src/apps/backend/modules/account/account_service.py

```diff
@@ -66,3 +68,15 @@ def get_account_by_username_and_password(*, params: AccountSearchParams) -> Acco
     @staticmethod
     def update_account_profile(*, account_id: str, params: UpdateAccountProfileParams) -> Account:
         return AccountWriter.update_account_profile(account_id=account_id, params=params)
+
+    @staticmethod
+    def create_or_update_account_notification_preferences(
+        *, account_id: str, preferences: NotificationPreferencesParams
+    ) -> NotificationPreferencesParams:
```

Comment:
nit: `NotificationPreferencesParams` - Why do we have `Params` at the end? It may make sense when bundling add, edit params in a dataclass, but when thinking OOPs, its not a good candidate for model name.

It also seems odd that input and output type is same.